pkg/kubelet/eviction/helpers_test.go

```diff
@@ -648,6 +733,47 @@ func TestAddAllocatableThresholds(t *testing.T) {
 	}
 }
 
+func TestFallbackResourcesUsage(t *testing.T) {
+	for _, test := range []struct {
+		description   string
+		usageFuncName string
+		usageFunc     func() int64
+	}{
+		{
+			description:   "disk usage, fallback value",
+			usageFuncName: "diskUsage",
+			usageFunc: func() int64 {
+				return diskUsage(&statsapi.FsStats{}).Value()
+			},
+		},
+		{
+			description:   "inode usage, fallback value",
+			usageFuncName: "inodeUsage",
+			usageFunc: func() int64 {
+				return inodeUsage(&statsapi.FsStats{}).Value()
+			},
+		},
+		{
+			description:   "memory usage, fallback value",
+			usageFuncName: "memoryUsage",
+			usageFunc: func() int64 {
+				return memoryUsage(&statsapi.MemoryStats{}).Value()
+			},
+		},
+		{
+			description:   "process usage, fallback value",
+			usageFuncName: "processUsage",
+			usageFunc: func() int64 {
+				return int64(processUsage(&statsapi.ProcessStats{}))
+			},
+		},
+	} {
+		t.Run(test.description, func(t *testing.T) {
+			assert.NotEqual(t, 0, test.usageFunc(), "unexpected fallback value")
```

Comment:
I don't see how `test.usageFuncName` is used in the test. It seems that tests only calls `test.usageFunc` and expects it to return non-zero result.

Reply:
my bad, resolved in 4632fc8a0f3